.github/workflows/test.yml

```diff
@@ -1,8 +1,8 @@
 name: Test
-
+ 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, crashitem ]
```

Comment:
This change is not needed. To run the PR in this repository (not the forks) I have to approve the run. This was introduced by GitHub to prevent crypto mining.

Reply:
I will revert it. It was used to run it on my own fork (couldn't test it on this one because of the approvals).